components/runtimes/charliecloud/SPECS/charliecloud.spec

```diff
@@ -99,8 +104,45 @@ EOF
 set     ModulesVersion      "%{version}"
 EOF
 
+cat > README.EL7 <<EOF
+For RHEL7 you must increase the number of available user namespaces to a non-
+zero number (note the number below is taken from the default for RHEL8):
+
+  echo user.max_user_namespaces=3171 >/etc/sysctl.d/51-userns.conf
+  reboot
+
+Note for versions below RHEL7.6, you will also need to enable user namespaces:
+
+  grubby --args=namespace.unpriv_enable=1 --update-kernel=ALL
+  reboot
+EOF
+
+cat > README.TEST <<EOF
+Charliecloud comes with a fairly comprehensive Bats test suite. For testing
+instructions visit: https://hpc.github.io/charliecloud/test.html
+EOF
+
 %{__mkdir_p} ${RPM_BUILD_ROOT}/%{_docdir}
 
+%check
+# Don't try to compile python files with /usr/bin/python
+%{?el7:%global __python %__python3}
+
```

Comment:
The `%check` section seems useless.

Reply:
Removed the `%check` section in my local branch. I've also moved the global python macro to the top. I believe you guys have this covered in `OHPC_macros` but we'd like to keep it.